haml/_error_summary.html.haml

```diff
@@ -1,6 +1,11 @@
-.cads-error-summary{"aria-live": "assertive"}
-  %p.cads-heading-small
+.cads-error-summary{"aria-live" => "assertive", "aria-labelledby" => "error-summary-title", :role => "alert", :tabindex => 0}
+  %p.cads-heading-small{:id => "error-summary-title"}
     There is a problem with
     %span.cads-error-summary__field_count 2 fields
   %ul.cads-list__no-bullet.cads-error-summary__list
-    %li Item 1
+    %li 
+      %a{:href => "#"}
```

Comment:
These links have no href, should they? Would they normally link to the field with an error?

Reply:
They should give focus to the erroring field (I put this in the usage notes section for this component).  I left it out here as I figured it was an implementation detail (eg I am doing it in React in the Email This Page component, but that might not be the case for other consumers).

I could make it more explicit here tho, if you think that would be useful? eg `<a href="#give-focus-to-input">...</a>`